modules/viz/include/opencv2/viz/viz3d.hpp

```diff
@@ -224,6 +234,26 @@ namespace cv
              */
             void spinOnce(int time = 1, bool force_redraw = false);
 
+            /** @brief Create a window in memory instead of on the screen.
+             */
+            void setOffScreenRendering();
+
+            /** @brief Remove all lights from the current scene.
+            */
+            void removeAllLights();
+
+            /** @brief Add a light in the scene.
+
+            @param position The position of the light.
+            @param focalPoint The point at which the light is shining
+            @param color The color of the light
+            @param diffuseColor The diffuse color of the light
+            @param ambientColor The ambient color of the light
+            @param specularColor The specular color of the light
+             */
+            void addLight(Vec3d position, Vec3d focalPoint = Vec3d(0, 0, 0), Color color = Color(255, 255, 255),
+                          Color diffuseColor = Color(255, 255, 255), Color ambientColor = Color(0, 0, 0), Color specularColor = Color(255, 255, 255));
+
```

Comment:
Would it be better to use Color::white() and Color::black() presets as in all other places of Viz?